resolwe_bio/tests/unit/test_api.py

```diff
@@ -2,7 +2,7 @@
 from rest_framework import status
 from rest_framework.test import APIRequestFactory, force_authenticate
 
-from resolwe.flow.models import Collection, Data, DescriptorSchema, Entity, Process
```

Comment:
Removing the descriptor should go into a separate commit.

Reply:
I have removed this file from the commit, since decriptor & descriptor_schema fields are already removed in the master branch.